Feature_time_series_db_cleanup/Feature_time_series_db_cleanup.md

```diff
@@ -0,0 +1,29 @@
+# Feature: Time series database cleanup
+
+## Problem
+
+Omotes 'forgets' about jobs when they are completed and the results have been retrieved.
+Except for the time series data which are stored in a database, and the results are kept
+indefinitely. This will cause the database to grow without an option to clean up, since omotes does
+not know which data is still relevant.
+Most front-ends will have an option to delete omotes runs which should also remove the time series
+data. Currently, this is possible by directly accessing the times series database. This is a bit

```

Comment:
Just to check. So currently this delete action is implemented in MapEditor without calling an OMOTES REST endpoint? DB credentials and delete logic are kept in ME code?

Reply:
A `omotes-rest` DELETE endpoint is called by the mapeditor.  
But currently this only removes the entry from the omotes-rest postgres db.